project/build.properties

```diff
@@ -1 +1,3 @@
-sbt.version=0.13.16
\ No newline at end of file
+#sbt.version=0.13.16
+#sbt.version=1.2.7
+sbt.version=1.1.1
```

Comment:
are the comments necessary?

Reply:
noob question...is scala 2.11 works with sbt 1.1?

Reply:
No I will remove it. Thanks for the point. 
Afaik scala 2.11 works on sbt 1.1. 
Anyway there is one issue i'm experiencing involving the cross publishing. It seems that scala 2.11 is ignored and only a build against scala 2.12 is executed